.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: feature request, needs triage
+assignees: ''
+---
+
+**Description:**
+Describe your proposal.
```

Comment:
I would add a note here to check for existing issues that have already been opened. This could hopefully reduce the amount of duplicate issues that come in.

The goal here is to have users first check for existing issues first before opening up a new issues